ml/src/main/scala/ai/h2o/sparkling/ml/backend/H2OModel.scala

```diff
@@ -15,7 +15,7 @@
 * limitations under the License.
 */
 
-package ai.h2o.sparkling.model
+package ai.h2o.sparkling.ml.backend
```

Comment:
the name `ai.h2o.sparkling.ml.backend` is very similar to `ai.h2o.sparkling.backend`. Would it be possible to use a more distinctive name? E.g. : `ai.h2o.sparkling.ml.internals` WDYT?

Reply:
Yup, good idea, like it 👍